contracts/utils/StrikeSelection.sol

```diff
@@ -77,7 +77,10 @@ contract StrikeSelection is DSMath, Ownable {
         //   with certain margin of error
         //        return strike price
 
-        uint256 strike = assetPrice.sub(assetPrice % step);
+        uint256 strike =
+            isPut
+                ? assetPrice.sub(assetPrice % step)
+                : assetPrice.add(step - (assetPrice % step));
```

Comment:
why do we do this?

Reply:
4.36. StrikeSelection, line 80: Note, `strike` should start with increased price in case of
a call.